packages/ListBox/src/components/NoResults/NoResults.js

```diff
@@ -13,7 +13,13 @@ export default function NoResults(props) {
   const [state] = useListBox();
 
   if ((state.isOpen || state.isInline) && state.noResultsFound) {
-    return <sc.NoResults data-pka-anchor="no-results">{label}</sc.NoResults>;
+    return (
+      <>
+        <sc.NoResults aria-live="polite" role="alert" data-pka-anchor="no-results">
```

Comment:
```suggestion
        <sc.NoResults role="status" data-pka-anchor="list-box.no-results">
```

Reply:
`alert` / `status` roles will also set the `aria-live` property.

Reply:
But actually, since this is conditionally rendered, it may have issue (though not in VoiceOver).  Maybe an invisible `<Toast>` is a better idea?